client/state/current-user/test/selectors.js

```diff
@@ -98,6 +99,48 @@ describe( 'selectors', () => {
 		} );
 	} );
 
+	describe( '#getCurrentUserIsRTL', () => {
+		it( 'should return null if the current user is not set', () => {
+			const isRTL = getCurrentUserIsRTL( {
+				currentUser: {
+					id: null
+				}
+			} );
+
+			expect( isRTL ).to.be.null;
+		} );
+
+		it( 'should return null if the current user isRTL slug is not set', () => {
```

Comment:
You probably meant `isRTL property` or `isRTL attribute` instead of `isRTL slug`?